src/components/ListingModal/listingModal.js

```diff
@@ -9,7 +9,7 @@ import FieldTag from "components/FieldTag/fieldTag";
 import { useNavigate } from "react-router-dom";
 
 const ListingModal = (props) => {
-  const { onHide, data } = props;
+  const { onHide, data, checkUser } = props;
```

Comment:
(minor issue)
was abit confused on what is `checkUser`
realised it was probably to decide whether or not to render the two buttons in the ListingModal footer
might be good to add a comment when we are cleaning up code